lib/apiservers/engine/proxy/common.go

```diff
@@ -15,5 +15,5 @@
 package proxy
 
 const (
-	swaggerSubstringEOF = "EOF"
+	SwaggerSubstringEOF = "EOF"
```

Comment:
Should this live in the constants package? Fine if there's a reason it's not there.

Reply:
It's not really a VIC constant.  I think we should keep swagger specific constants and code cordon to its own area.